docs/smart-contracts/serving-nft-metadata.md

```diff
@@ -63,7 +63,7 @@ By default, the `tokenURI()` method will take this URI and append the requested
 
 ```tsx
 tokenURI(1) { return _baseURIString + '/' + 1 }
-// https://testnets.tableland.network/query?unwrap=true&extract=true&statement=/1
+// https://testnets.tableland.network/query?unwrap=true&extract=true&statement=1
```

Comment:
@dtbuchholz this is obviously correcting a typo, but also noticing that the path is wrong here (maybe that's on purpose?).
should `https://testnets.tableland.network/query` be `https://testnets.tableland.network/api/v1/query`?

Reply:
We can address that in a different PR, just figured I'd mention it.

Reply:
good catch, updated in my next docs batch!